src/components/ValueCard/_unit-renderer.scss

```diff
@@ -0,0 +1,25 @@
+@import '../../globals/vars';
+
+.#{$iot-prefix}--value-card__attribute-unit {
+  padding-left: 0.25rem;
+  font-weight: lighter;
+  white-space: nowrap;
+  padding-bottom: 0.25rem;
+  font-size: var(--default-font-size);
+}
+
+.#{$iot-prefix}--value-card__attribute-unit--mini {
+  padding-bottom: 0;
+  font-size: 1rem;
```

Comment:
Needs carbon styling variables

Reply:
Done. 
Can I just assume that it is always safe translate 1rem into $carbon--spacing-05 or are there any special considerations needed given that the 1rem value of $carbon--spacing-05 is defined as a SCSS default and can be overridden?

Reply:
As far as I've seen / know, its safe to stick to those variables. I've personally never seen them change, but @davidicus or @tay1orjones could maybe give a better answer

Reply:
Yes, typically it's a safe bet to use `$carbon--spacing-05` in lieu of `1rem` on layout properties (`padding`, `margin`, `top`, etc). Though it's worth considering if the variable value should change, how that may impact or break the component. By and large we should be using the variables and avoid hardcoded values.

In this case though, font settings should generally be defined by `@include`ing the [typography variables](https://www.carbondesignsystem.com/guidelines/typography/productive), not spacing variables. In this case we might want to use `productive-heading-02`?

`@include type-style('productive-heading-02');`